begin.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Apple solo project</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" type="text/css" href="reset.css" />
+    <link rel="stylesheet" type="text/css" href="begin.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <style>
+      A {
+        text-decoration: none;
+      }
+    </style>
```

Comment:
[REQUIRED] Let's remove this CSS from the HTML file. You can add it to the CSS file if needed.

Reply:
Done